validator/project_validator.go

```diff
@@ -1290,7 +1290,10 @@ func validatePluginCommands(project *model.Project) ValidationErrors {
 	}
 
 	if project.EarlyTermination != nil {
```

Comment:
I tried removing it and it seems like the validator didn't produce any warning or error for it. I removed it entirely.

Reply:
Cool! I'm a bit torn actually; we may want to notify relevant projects that we're removing it if they _do_ want to declutter their yamls. UnmarshalStrict is supposed to catch things like this (assuming you aren't running it with quiet), but if it isn't then we _should_ maybe keep the warning for a little bit and then remove in a bit so people have time to clean up if they want (and/or we can make a ticket to verify that UnmarshalStrict is working correctly since that would also do this)

Reply:
Whichever you decide, LGTM